spec/mongoid/interceptable_spec.rb

```diff
@@ -2594,4 +2594,37 @@ class TestClass
       end
     end
   end
+
+  context "when around callbacks for embedded children are enabled" do
+    config_override :around_callbacks_for_embeds, true
+
+    context "when around callback is defined without a yield" do
+      class Foo
+        include Mongoid::Document
+        embeds_many :bars, cascade_callbacks: true
+      end
+
+      class Bar
+        include Mongoid::Document
+        embedded_in :foo
+        field :flag_around_save, type: String
+        around_save :log_callback
+
+        private
+
+        def log_callback
+          flag_around_save = true
+        end
+      end
+
+      foo = Foo.create
+      2.times { foo.bars.build }
```

Comment:
If you declare a value at the `context` level like this, it will be evaluated once for the entire context. In this case, it doesn't matter (because there's only one test), but in general you want the value to be "clean" for each test. To accomplish this, rspec provides a `let` method to declare these variables in a way that allows you to re-evaluate them for each test, e.g.:

```suggestion
      let(:foo) do
        Foo.create.tap do |foo|
          2.times { foo.bars.build }
        end
      end
```

We can make it a bit more concise by providing the `bars` list to the `#create` call, like this:

```suggestion
      let(:foo) { Foo.create(bars: [ Bar.new, Bar.new ]) }
```